test/integration/crcsuite/crcsuite.go

```diff
@@ -73,11 +78,71 @@ func FeatureContext(s *godog.Suite) {
 		DeleteFileFromCRCHome)
 
 	s.BeforeSuite(func() {
-		// set CRC home var
-		CRCHome = SetCRCHome()
+
+		usr, _ := user.Current()
+		CRCHome = filepath.Join(usr.HomeDir, ".crc")
+
+		// init CRCBinary if no location provided by user
+		if CRCBinary == "" {
+			fmt.Println("Expecting the CRC binary to be in $HOME/go/bin.")
+			usr, _ := user.Current()
+			CRCBinary = filepath.Join(usr.HomeDir, "go", "bin")
+		}
+
+		// put CRC binary location on top of PATH
+		path := os.Getenv("PATH")
+		var separator string
+		switch platform := runtime.GOOS; platform {
+		case "darwin":
+			separator = ":"
```

Comment:
There is already `os.PathListSeparator` so you shouldn't need to manually define it.

Reply:
Thanks! I only found `filepath.SplitList` method. This is useful.